### ISSUE TITLE
oom() compile error (nightly change)

### ISSUE THREAD

**Issue by MarkSwanson** [Amanieu/hashmap_core#3]
Just an fyi...

**Comment by MarkSwanson** [comment]
fyi this 'fixed' it for me: added to lib.rs:
```
+#![feature(lang_items)]
+#[lang = "oom"]
+fn oom() -> ! {
+    panic!("oom")
+}
+

In src/map.rs and src/table.rs I did things like:
+use oom;
-            Err(CollectionAllocErr::AllocErr) => Global.oom(),
+            Err(CollectionAllocErr::AllocErr) => oom(),

```

**Comment by Amanieu** [comment]
Fixed by #10.
